src/main/java/Loafy/command/FindCommand.java

```diff
@@ -0,0 +1,17 @@
+package loafy.command;
+
+import loafy.tasklist.TaskList;
+import loafy.ui.Ui;
+
+public class FindCommand extends Command {
```

Comment:
I think you might have missed out this file while doing the JavaDoc. Just to let you know ;)